Makefile

```
@@ -317,12 +331,21 @@ setup-dev: ##@setup Install all necessary tools for development
 setup-dev:
 	echo "Replaced by Nix shell. Use 'make shell' or just any target as-is."
 
+USE_SYSTEM_NIM ?= 0
+
+generate-sds:  ##@ Build libsds third_party
```

This does not really "generate", right?

```suggestion
build-sds:  ##@ Build libsds third_party
```